oak_containers_hello_world_container/build_container_bundle

```diff
@@ -15,8 +15,13 @@ cargo build \
     -Zunstable-options \
     --out-dir='./target/'
 
-docker build --tag="${CONTAINER_IMAGE}" .
+# Export the container as an OCI Image. Ref:
```

Comment:
Is the ref missing?

Reply:
good catch! classic juliette kind of slip up. will add :)